plugins/importexport/native/filter/NativeFilterHelper.inc.php

```diff
@@ -83,32 +122,100 @@ function createCoversNode($filter, $doc, $object) {
+						$coverImage = $this->parsePublicationCover($filter, $n, $object);
+						$coverImages[key($coverImage)] = reset($coverImage);
 						break;
 					default:
-						$deployment->addWarning($assocType, $object->getId(), __('plugins.importexport.common.error.unknownElement', array('param' => $n->tagName)));
+						$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.unknownElement', array('param' => $n->tagName)));
 				}
 			}
 		}
+
+		$object->setData('coverImage', $coverImages);
+	}
+
+	/**
+	 * Parse out the object covers.
+	 * @param $filter NativeExportFilter
+	 * @param $node DOMElement
+	 * @param $object Issue
+	 */
+	function parseCovers($filter, $node, $object) {
+		$deployment = $filter->getDeployment();
+		for ($n = $node->firstChild; $n !== null; $n=$n->nextSibling) {
+			if (is_a($n, 'DOMElement')) {
+				switch ($n->tagName) {
+					case 'cover':
+						$this->parseCover($filter, $n, $object);
+						break;
+					default:
+						$deployment->addWarning(ASSOC_TYPE_ISSUE, $object->getId(), __('plugins.importexport.common.error.unknownElement', array('param' => $n->tagName)));
+				}
+			}
+		}
+	}
+
+	/**
+	 * Parse out the cover and store it in the object.
+	 * @param $filter NativeExportFilter
+	 * @param $node DOMElement
+	 * @param $object Publication
+	 */
+	function parsePublicationCover($filter, $node, $object) {
+		$deployment = $filter->getDeployment();
+
+		$context = $deployment->getContext();
+
+		$locale = $node->getAttribute('locale');
+		if (empty($locale)) $locale = $context->getPrimaryLocale();
+
+		$coverImagelocale = array();
+		$coverImage = array();
+
+		for ($n = $node->firstChild; $n !== null; $n=$n->nextSibling) {
+			if (is_a($n, 'DOMElement')) {
+				switch ($n->tagName) {
+					case 'cover_image': 
+						$coverImage['uploadName'] = $n->textContent; 
+						break;
+					case 'cover_image_alt_text':
+						$coverImage['altText'] = $n->textContent; 
+						break;
+					case 'embed':
+						import('classes.file.PublicFileManager');
+						$publicFileManager = new PublicFileManager();
+						$filePath = $publicFileManager->getContextFilesPath($context->getId()) . '/' . $coverImage['uploadName'];
+						file_put_contents($filePath, base64_decode($n->textContent));
+						break;
+					default:
+						$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.unknownElement', array('param' => $n->tagName)));
+				}
+			}
+		}
+
+		$coverImagelocale[$locale] = $coverImage;
+
+		return $coverImagelocale;
 	}
 
 	/**
 	 * Parse out the cover and store it in the object.
 	 * @param $filter NativeExportFilter
 	 * @param $node DOMElement
-	 * @param $object Issue|Article
-	 * @param $assocType ASSOC_TYPE_ISSUE | ASSOC_TYPE_SUBMISSION
+	 * @param $object Issue
 	 */
-	function parseCover($filter, $node, $object, $assocType) {
+	function parseCover($filter, $node, $object) {
```

Comment:
Likewise, maybe this should be `parseIssueCover`?

Reply:
Fixed.